products/motorola-mobility.md

```diff
@@ -19,6 +19,19 @@ eolColumn: Security Updates
 #
 
 releases:
+- releaseCycle: "moto-e15"
+  releaseLabel: "Motorola Moto E15"
+  releaseDate: 2024-12-01
+  eol: 2026-12-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/#gs=eyJndWlkZUlEIjo3MTEyLCJxdWVzdGlvbklEIjo1LCJyZXNwb25zZUlEIjoxNzQsImd1aWRlU2Vzc2lvbiI6ImlJYm5jbzNyIiwic2Vzc2lvbklEIjoiNExRV2FvM3IifQ..
+
+- releaseCycle: "moto-e14"
+  releaseLabel: "Motorola Moto E14"
+  releaseDate: 2024-04-01
+  eol: 2026-04-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/#gs=eyJndWlkZUlEIjo3MTEyLCJxdWVzdGlvbklEIjo1LCJyZXNwb25zZUlEIjoxMjgsImd1aWRlU2Vzc2lvbiI6ImlJYm5jbzNyIiwic2Vzc2lvbklEIjoiNExRV2FvM3IifQ..
```

Comment:
```suggestion
  link: https://en-us.support.motorola.com/app/product_page/faqs/p/10216
```

The current link is a generic page. I also did not found a software-security-update page .